notebook/static/notebook/js/keyboardmanager.js

```diff
@@ -28,21 +28,36 @@ define([
          *    @param options.events {$(Events)} instance 
          *    @param options.pager: {Pager}  pager instance
          */
-        this.mode = 'command';
         this.enabled = true;
         this.pager = options.pager;
         this.quick_help = undefined;
         this.notebook = undefined;
         this.last_mode = undefined;
+		this.current_mode = "command";
         this.bind_events();
         this.env = {pager:this.pager};
         this.actions = options.actions;
-        this.command_shortcuts = new keyboard.ShortcutManager(undefined, options.events, this.actions, this.env );
```

Comment:
Yes of course ... but I havn't yet precisely to track all occurence of old API through tests. I 'll do that at the end.